falcon/media/msgpack.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+
+from falcon import errors
+from falcon.media import BaseHandler
+
+
+class MessagePackHandler(BaseHandler):
+    """Handler built using the :py:mod:`msgpack` module from python-msgpack
+
```

Comment:
[fix] Add a note saying that this handler assumes that clients encode using the bin type to disambiguate between strings and bytes.